app/templates/ember-community-survey-2017.hbs

```diff
@@ -17,19 +17,10 @@
   @sectionTitle="Ember Version"
 >
   <:charts>
-    <div>
-      <Highcharts
-        @chartOptions={{this.chartHowlong.options}}
-        @data={{this.chartHowlong.data}}
-      />
-    </div>
-
-    <div class="mt-3">
-      <Highcharts
-        @chartOptions={{this.chartBreakdownByVersion.options}}
-        @data={{this.chartBreakdownByVersion.data}}
-      />
```

Comment:
I removed the second chart because the new spline chart (`chartHowlong`) displays the same data as the bar chart (`chartBreakdownByVersion`), but is easier to read and contains one additional year of data.